Source/WebCore/rendering/RenderObject.h

```diff
@@ -789,9 +787,10 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
         ApplyCompositedClips = 1 << 1,
         ApplyCompositedContainerScrolls  = 1 << 2,
         ApplyContainerClip = 1 << 3,
+        CalculateFastRepaintRect = 1 << 4,
```

Comment:
I am sorry I did not search the code before making this suggestion. I think we should go for the opposite: `CalculateAccurateRepaintRect`. This will make less changes and it makes sense because it seems most of the callers of the VisibleRectContextOption need fast `repaintRect`.

Reply:
Changed.